src/uanodeid.cpp

```diff
@@ -83,6 +93,20 @@ UaString UaNodeId::toString() const
     return "non-string-id";
 }
 
+void UaNodeId::copyTo( UaNodeId* other) const
+{
+    *other = *this;
+}
+
+/** Assumption: the other is "initialized"
```

Comment:
no way to tell if other is initialised?

Reply:
handled the case of nullptr; the other part should be implemented by UA_NodeId_copy which return value gets checked. Removed the mention of 'assumption'.